cmd/crossbuild.go

```diff
@@ -135,7 +135,7 @@ func runCrossbuild() {
 			dockerPowerPCBuilderImage, "-i", repoPath, "-p", q(powerPCPlatformsParam))
 	}
 
-	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" {
+	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" && version.Compare(goVersion, "1.6", ">=") {
```

Comment:
So the minimum version required for mips is 1.6? Shouldn't that be communicated to the user when they try to build a project with go1.5 and specify mips?